pandas/tests/indexes/datetimes/test_arithmetic.py

```diff
@@ -740,6 +740,17 @@ def test_sub_period(self, freq):
         with pytest.raises(TypeError):
             p - idx
 
+    @pytest.mark.parametrize('op', [operator.add, ops.radd,
```

Comment:
can you add (maybe another test) add/sub with a pi and another dtype (I don't think we have this test?)

Reply:
Just checking, you mean to add cases over in the period test_arithmetic file, not here, right?